realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1934,6 +1935,231 @@ public boolean isValid() {
         return this;
     }
 
+    /**
+     * Condition that value of field contains the specified key.
+     *
+     * @param fieldName the field to compare.
+     * @param key       the key to search for.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException if the field name isn't valid.
+     */
+    public RealmQuery<E> containsKey(String fieldName, @Nullable String key) {
+        realm.checkIfValid();
+        this.query.containsKey(fieldName, Mixed.valueOf(key));
+        return this;
+    }
+
+    /**
+     * Condition that value of field contains the specified value.
+     *
+     * @param fieldName the field to compare.
+     * @param value     the value to search for.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException if the field name isn't valid.
+     */
+    public RealmQuery<E> containsValue(String fieldName, @Nullable Boolean value) {
```

Comment:
We should make it clear this only works for Dictionaries (and I suspect we later want to use this for Sets as well?) Same for all the other methods